Readability.js

```diff
@@ -1722,6 +1722,23 @@ Readability.prototype = {
   isProbablyReaderable: function(helperIsVisible) {
     var nodes = this._getAllNodesWithTag(this._doc, ["p", "pre"]);
 
+    // Get <div> nodes which have <br> node(s) and append them into the `nodes` variable.
+    // Some articles' DOM structures might look like
+    // <div>
+    //   Sentences<br>
+    //   <br>
+    //   Sentences<br>
+    // </div>
+    var brNodes = this._getAllNodesWithTag(this._doc, ["div > br"]);
+    if (brNodes.length > 1) {
+      var set = new Set();
+      [].forEach.call(brNodes, function(node) {
+        var div = node.parentNode;
+        !set.has(div) && set.add(div);
+      });
+      nodes = [].concat.call(Array.from(set), nodes);
+    }
+
```

Comment:
Looks like the patch I provided here is not acceptable, and at same time we don't have better ideas. Maybe we need to re-design the `isProbablyReaderable` method[1] here.

As I know, Safari's reader mode is quite good, so maybe we could try to use some ideas from Safari algorithm[2] to design our new `isProbablyReaderable` method(even porting the `adoptableArticle` method from Safari algorithm?). I think it can help us to resolve the problem.

What do you think, @gijsk ?

[1]: https://github.com/mozilla/readability/blob/master/Readability.js#L1722-L1758
[2]: https://github.com/adadgio/safari-reader-js/blob/master/safari-reader.js#L1021-L1045